_posts/2023-12-13-help-beagle.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Help a Skater become a Builder
+description: Campaign to raise funds for BeagleXV
+date: 2023-12-13 00:00:00 +0000
+permalink: /help-beagle/
+author: vlad
+image: /images/gnars.png
+image_caption:
+video_embed: 
+tags: [research]
+featured: 
+---
+
+
+# Testing a Pull Request in Gnars.com , if it works I will complete the post immediatly 
+
+
+
+
+<div style="position:relative;width:100%;padding-top:calc(100% + 72px)">
+  <iframe src="https://zora.co/collect/zora:0xafc7733615cc76efe015f4830886098d5e78facd/1/embed?referrer=0x41CB654D1F47913ACAB158a8199191D160DAbe4A" style="border:0;background-color:transparent;position:absolute;inset:0" width="100%" height="100%" allowtransparency="true" allowfullscreen="true" sandbox="allow-pointer-lock allow-same-origin allow-scripts allow-popups"></iframe>
+</div>
+<a href="https://zora.co/collect/zora:0xafc7733615cc76efe015f4830886098d5e78facd/1" style="padding:12px;text-decoration:none;color:#b3b3b3;font-family:'Inter',system-ui;font-size:10px;line-height:12px;font-style:normal;font-weight:400">I have helped a skater to buy a computer. on Zora</a>
```

Comment:
idk if this was intentional but there is a typo here: `I have helped a skater to buy a computer on Zora.` period in the wrong place